tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
Instead of having a separate file for tests requirements, why not simply add the requirements here? Any advantages on having `tests_require` in setup.py?